src/server/json_family.cc

```diff
@@ -474,11 +474,17 @@ OpResult<JsonCallbackResult<T>> JsonEvaluateOperation(const OpArgs& op_args, std
   return json_path.Evaluate<T>(*result, cb, options.save_first_result);
 }
 
+struct MutateOperationOptions {
+  JsonReplaceVerify verify_op;
+  bool empty_is_nil = false;
+};
+
 template <typename T>
-OpResult<JsonCallbackResult<optional<T>>> UpdateEntry(const OpArgs& op_args, std::string_view key,
-                                                      const WrappedJsonPath& json_path,
-                                                      JsonPathMutateCallback<T> cb,
-                                                      JsonReplaceVerify verify_op = {}) {
+OpResult<JsonCallbackResult<optional<T>>> JsonMutateOperation(const OpArgs& op_args,
+                                                              std::string_view key,
+                                                              const WrappedJsonPath& json_path,
+                                                              JsonPathMutateCallback<T> cb,
```

Comment:
should `cb` be a member of `MutateOperationOptions`?

Reply:
I do not think so. Since this is the main argument in this function. This function is created actually to run this callback on some json value, . So I don't think it's just an option.